build.gradle

```diff
@@ -54,7 +54,11 @@ repositories {
 
 }
 
-final htsjdkVersion = System.getProperty('htsjdk.version','2.8.0')
+// TODO: switch to a non-snapshot version
+// (it's snapshot now because we rely on not-yet-released changes to htsjdk)
+//final htsjdkVersion = System.getProperty('htsjdk.version','2.8.1-6-g2acb88b-20170106.162419-1')
```

Comment:
No need to add these comments -- we are often on an htsjdk snapshot as a project.